### ISSUE TITLE
Get the observables into the ensemble results

### ISSUE THREAD

**Issue by jlperla** [SciML/DifferenceEquations.jl#63]
Right now the ensemble stuff works for getting the underlying dynamics (i.e., the `u`) but doesn't provide the `z` from the solutions.  Might be nice to have a way to get that in the ensemble simulations as well.  For example, a setting to concatenate them/etc.

**Comment by ChrisRackauckas** [comment]
Yeah, I noticed this. We need some way to make the ensembles a bit more general, for here and the optimization results.
